server/src/mcr/lookahead/lookaheadController.spec.js

```diff
@@ -4,13 +4,17 @@ const request = require('supertest')
 import LookaheadController from './lookahead-controller'
 import EhrApp from '../../server/app'
 import Config from '../../config/config'
+import Helper from '../common/test-helper'
 
 const config = new Config('test')
 const configuration = config.config
 const ehrApp = new EhrApp()
 const BASE = '/api/lookahead'
 const typeName = 'Lookahead'
 
+const visitId = Helper.sampleObjectId(true)
```

Comment:
Not used?

Reply:
Actually, it is, since I had previously added the auth middleware to the feedback end point. Therefore, to successfully pass the tests, it needs the authToken.